test/e2e/addonmanagement_test.go

```diff
@@ -48,7 +48,7 @@ var (
 	}
 )
 
-var _ = ginkgo.Describe("Enable addon management feature gate", ginkgo.Ordered, ginkgo.Label("addon-manager"), func() {
+var _ = ginkgo.FDescribe("Enable addon management feature gate", ginkgo.Ordered, ginkgo.Label("addon-manager"), func() {
```

Comment:
what does Fdescribe here mean?

Reply:
I am debuging the flaky e2e case, will change this back.